README.md

```diff
@@ -1,101 +1,246 @@
+     key.serializer = "org.apache.kafka.common.serialization.StringSerializer"
+     value.serializer = "org.apache.kafka.common.serialization.StringSerializer"
+  }
+```
+  
+* Step 3) Save a Twirl template that also will be persisted in Zookeeper.
+```
+  hermes> set template
+  Press Control + D to finish
+  @import com.stratio.hermes.utils.Hermes
+  @(hermes: Hermes)
+  {
+    "name" : "@(Hermes.Name.firstName)"
+  }
+```
+* Step four) Once you have saved these configuration in ZK, you could start a generation in the nodes that you need:
+```
+  hermes> ls
+  Node Id                                Status
+  845441ec-cb0d-4363-b494-a39d56a82727 | false
+  hermes> start 845441ec-cb0d-4363-b494-a39d56a82727
+  hermes> ls
+  Node Id                                Status
+  845441ec-cb0d-4363-b494-a39d56a82727 | true
+```
+  At this moment the node with id 845441ec-cb0d-4363-b494-a39d56a82727 is producing messages to Kafka following the saved template. You can check it using Kafka's console consumer.
+
+## Random Helper.
+Supporting us in [Faker](https://github.com/stympy/faker) we are developing a random generator. At this moment we have the next features:
+* Name generation:
+```
+  fullname() → Paul Brown
+  middleName() → George Michael
+  firstName() → Steven
+  lastName() → Robinson
+```
+
+* Number generation:
+```
+  number(2) → 23
+  number(2,Positive) → 23
+  decimal(2) → 23.45
+  decimal(2,Negative) → -45.89
+  decimal(2,4) → 45.7568
+  decimal(3,2,Positive) → 354.89
+  numberInRange(1,9) → 2
+  decimalInRange(1,9) → 2.6034840849740117
+```
+
+* Geolocation generation:
+```
+  geolocation() → (40.493556, -3.566764, Madrid)
+  geolocationWithoutCity() → (28.452717, -13.863761)
+  city() → Tenerife
+  country() → ES
+```
+
+* Timestamp generation:
+```
+  dateTime("1970-1-12" ,"2017-1-1") → 2005-03-01T20:34:30.000+01:00
+  time() → 15:30:00.000+01:00
+```
+* Music generation:
+```
+  playedSong() → ("Despacito (Featuring Daddy Yankee)", "Luis Fonsi, Daddy Yankee", "Despacito (Featuring Daddy Yankee)","Latin")
+```
+
+## Docker.
+* Seed + Node
+```sh
+  docker run -dit --name SEED_NAME -e PARAMS="-Dhermes.client=true -Dakka.remote.hostname=SEED_NAME.DOMAIN -Dakka.remote.netty.tcp.port=2552 -Dakka.remote.netty.tcp.hostname=SEED_NAME.DOMAIN -Dakka.cluster.seed-nodes.0=akka.tcp://hermes@SEED_NAME.DOMAIN:2552" qa.stratio.com/stratio/hermes:VERSION
+```
+* Node
+```sh
+  docker run -dit --name AGENT_NAME -e PARAMS="-Dhermes.client=false -Dakka.remote.hostname=AGENT_NAME.DOMAIN -Dakka.remote.netty.tcp.port=2553 -Dakka.cluster.seed-nodes.0=akka.tcp://hermes@SEED_NAME.DOMAIN:2552" qa.stratio.com/stratio/hermes:VERSION
+```
+  
+## FAQ.
+* **Is it neeeded Zookeeper to run KHermes?.** Yes, at this moment it is mandatory to have a instance of Zookeper in order to run KHermes.
+* **Is it needed Apache Kafka to run KHermes?.** Yes, at the end all generated event will be persisted in Kafka and right now there are not any other possibility.
```

Comment:
Is Apache Kafka needed ...? ... all generated events... there are no other possibilities.